src/frameworks/foundation/ns_timer.rs

```diff
@@ -79,7 +79,7 @@ pub const CLASSES: ClassExports = objc_classes! {
                             selector:(SEL)selector
                             userInfo:(id)user_info
                              repeats:(bool)repeats {
-    let timer = msg![env; this timerWithTimeInterval:interval
+    let timer: id = msg![env; this timerWithTimeInterval:interval
```

Comment:
Please fix the indentation of the subsequent lines if you're going to change this. It should already be inferred as `id` due to the return type so just reverting it is also an option.

Reply:
For some reason, it doesn't infer the type properly (on rustc 1.71.1). I'll fix the indentation though.